packages/lxd/src/file/pull.coffee.md

```diff
@@ -0,0 +1,114 @@
+# `nikita.lxc.file.pull`
+
+Pull files from containers.
+
+## Example
+
+```js
+const {$status} = await nikita.lxc.file.pull({
+  container: 'my_container',
+  source: './a_file',
+   target: `#{scratch}/a_file`
+})
+console.info(`File was pulled: ${$status}`)
+```
+
+## Todo
+
+# Allow the automatic creation of directory `create_dirs` property
+## Schema definitions
+
+    definitions =
+      config:
+        type: 'object'
+        properties:
+          'algo':
+            default: 'md5'
+            $ref: 'module://@nikitajs/core/lib/actions/fs/hash#/definitions/config/properties/algo'
+          'container':
+            $ref: 'module://@nikitajs/lxd/src/init#/definitions/config/properties/container'
+            description: '''
+            Name of the container in lxd.
+            '''
+          'create_dirs':
+            type: 'boolean'
+            default: false
+            description: '''
+            # Local machine side 
```

Comment:
Remove md format or whatever the `# ...` means, plain english sentences are expected for description